client/components/Nav.tsx

```diff
@@ -1,16 +1,167 @@
-import React from "react";
-import { NavLink } from "react-router-dom";
+import React, {useState} from "react";
+import { NavLink, useNavigate} from "react-router-dom";
+import AppBar from "@mui/material/AppBar";
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+import IconButton from '@mui/material/IconButton';
+import Box from '@mui/material/Box';
+import MenuIcon from '@mui/icons-material/Menu';
+import AccountCircle from '@mui/icons-material/AccountCircle';
+import SendIcon from '@mui/icons-material/Send'
+import MenuItem from '@mui/material/MenuItem';
+import Menu from '@mui/material/Menu';
+import { selectCurrentUser } from "../features/authSlice";
 
-type NavProps = {}
 
-const Nav = (props: NavProps) =>  {
-    return(
-        <nav>
+const Nav = () =>  {
+    const [menuIconAnchorEl, setMenuIconAnchorEl] = useState<null | HTMLElement>(null);
+    const [accountCircleAnchorEl, setAccountCircleAnchorEl] = useState<null | HTMLElement>(null);
+
+    const navigate = useNavigate();
+    const isLoggedIn = !!selectCurrentUser;
```

Comment:
This will always be false, because currently the authSlice is not storing the user after register/login. Might be worth filing a bug to get that fixed in a separate PR. But you won't be able to test this until logged in user is properly loaded into the store

Reply:
I think we added `state.user` to the send a zoop feature PR right?